uniflow/op/prompt.py

```diff
@@ -57,6 +57,6 @@ class PromptTemplate(BaseModel):
 
     instruction: str = Field(..., min_length=0)
 
-    few_shot_prompt: conlist(Context, min_length=0) = Field([], min_items=0)
+    few_shot_prompt: conlist(Context) = Field([])
```

Comment:
I want to make sure if changing this few_shot_prompt is the correct approach. In my case, I am encountering the error `TypeError: conlist() got an unexpected keyword argument 'min_length'` if I did not change this code

Reply:
This makes sense to me. I changed this to min_length=0 due to feature request that user want to use zero shot.

Reply:
Got it :)